crypto/benches/crypto.rs

```diff
@@ -134,6 +145,9 @@ mod ed25519_benches {
         c.bench_function("BLS12381 keypair generation", move |b| {
             b.iter(|| BLS12381KeyPair::generate(&mut csprng))
         });
+        c.bench_function("Secp256k1 keypair generation", move |b| {
+            b.iter(|| Secp256k1KeyPair::generate(&mut csprng))
+        });
     }
 
     criterion_group! {
```

Comment:
Not that this is part of your PR, but this criterion group needs a better name 😁

Reply:
Yes let's do in an upcoming cleanup PR